### PR TITLE
[RW-5672][risk=low]  API: Create 4 "fake" concept sets to support 4 fitbit tables

### PR DESCRIPTION
**Pull request by NehaBroad**
- getValues for all 4 fitbit Domains

- get preview data for all 4 Fitbit domains

- Generate Code for all 4 **fitibit** Domains

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Review by freemabd — COMMENTED** [review]
At first glance this all looks fine. Can we add a full integration test that uses one of the prepackaged fitbit tables like https://github.com/all-of-us/workbench/blob/55a18a803d11dc748301590483b2a2b3c1c49146/api/src/bigquerytest/java/org/pmiops/workbench/api/DataSetControllerBQTest.java#L480-L480

File: api/src/main/resources/workbench-api.yaml [hunk 1]
```diff
@@ -4841,6 +4845,11 @@ definitions:
     - NONE
     - PERSON
     - SURVEY
+    - FITBIT
+    - FITBIT_HEART_RATE_SUMMARY
+    - FITBIT_HEART_RATE_LEVEL
+    - FITBIT_ACTIVITY
+    - FITBIT_INTRADAY_STEPS
     - BOTH
```
Comment (freemabd) [inline api/src/main/resources/workbench-api.yaml:4853]: So if i select multiple of these prepackaged concept sets, how does the request/API handle this? The request only carries an enum value currently.

Comment (NehaBroad) [reply api/src/main/resources/workbench-api.yaml:4853]: I will be handling multiple PRE-PACKAGED concept set in the ticket RW-5687 along with UI changes

Comment (freemabd) [reply api/src/main/resources/workbench-api.yaml:4853]: Gotcha

**Review by freemabd — APPROVED** [review]
Thanks for adding the test.

**Pull request merged**